src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2854,3 +2854,11 @@ Include=Include
 Exclude=Exclude
 Include\ or\ exclude\ cross-referenced\ entries=Include or exclude cross-referenced entries
 Would\ you\ like\ to\ include\ cross-reference\ entries\ in\ the\ current\ operation?=Would you like to include cross-reference entries in the current operation?
+
+New\ Library=New Library
```

Comment:
Very strange that a new localization is needed.

Please rework the strings to be in line with the other casing in JabRef

![grafik](https://github.com/user-attachments/assets/2bac884a-d3aa-4d0f-a0e0-6c12b35fa0c6)